modules/node-tuning-operator.adoc

```diff
@@ -3,9 +3,26 @@
 // * scalability_and_performance/using-node-tuning-operator.adoc
 
 [id='about-node-tuning-operator-{context}']
-= Understanding the node tuning operator
+= About the Node Tuning Operator
 
-The node tuning operator helps you manage node-level tuning by orchestrating the
-tuned daemon.
+The Node Tuning Operator helps you manage node-level tuning by orchestrating the
+tuned daemon. The Operator:
 
-Here are more details about why it is used...
+* Provides a basic, out-of-the box system tuning in the {product-title} cluster.
+* Ensures that the applications shipped with {product-title} 4.0 and later that
+have different tuning needs and default settings run without issues.
```

Comment:
different tuning needs **and** default settings?  What I was trying to say originally was that if you have an OCP 4.0-shipped application that will not run with default OS (RHCoS/RHEL) settings, the operator will make sure it applies the necessary OS-level settings.  For example Elasticsearch 5.  Please see: https://github.com/openshift/cluster-node-tuning-operator/blob/master/assets/tuned/07-cr-tuned.yaml#L74 for an example.